samcli/commands/package/command.py

```diff
@@ -112,8 +131,20 @@ def do_cli(
     metadata,
     region,
     profile,
+    resolve_s3,
 ):
     from samcli.commands.package.package_context import PackageContext
+    from samcli.lib.bootstrap.bootstrap import manage_stack
+    from samcli.commands.deploy import exceptions as deploy_exceptions
+
+    if resolve_s3:
+        if s3_bucket:
```

Comment:
`s3_bucket` is currently a required parameter. You'll need to make it no longer required, and add a check that either `s3_bucket` or `resolve_s3` is specified. That might also be the right place to check that you raise an error if both are specified.